_posts/static/2018-02-07-use-case.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+howtos: true
+published: true
+title: Use case
+permalink: howtos/use-case
+description: Features 
+---
+
+## Registration feature
+
+The Flyve MDM Demo plugin, allows the self creation of accounts, feature currently not available from GLPI, this means that users who don't have a previous account in GLPI can be able to generate their own accounts from the Web App without requesting these credentials to a registered user.
+
+## Time trial
+
+Demo plugin comes with a time limit trial of 90 days, if this time limit it's not enabled, the users' accounts created and validated will have access to the features provided either in Flyve MDM nor GLPI, however if enabled the accounts registered will be disabled after this period of time.
```

Comment:
I suggest the following:

The demo plugin comes with a time limit trial of 90 days, if this time limit is not enabled, the user's accounts created and validated will have access to the features provided either in Flyve MDM nor GLPI without diration limit, however if enabled the accounts registered will be disabled after this period of time.

Reply:
Changing ...